pkg/egressgateway/endpoint.go

```diff
@@ -7,10 +7,10 @@ import (
 	"fmt"
 	"net"
 
-	"k8s.io/apimachinery/pkg/types"
-
 	k8sTypes "github.com/cilium/cilium/pkg/k8s/types"
 	"github.com/cilium/cilium/pkg/labels"
+
+	"k8s.io/apimachinery/pkg/types"
```

Comment:
Nit: I think the order of the import groups should stay as is, i.e. 1) standard library, 2) third-party and 3) local packages. See #18357 for context.